src/app/components/core/asset-readings/series-graph/series-graph.component.html

```diff
@@ -0,0 +1,47 @@
+<div id="series_graph" class="modal">
+  <div class="modal-background"></div>
+  <div class="modal-card">
+    <header class="modal-card-head">
+      <p class="modal-card-title is-size-5">{{assetCode}}
+      </p>
+      <div class="close">
+        <button class="delete" aria-label="close" (click)="toggleModal(false)"></button>
+      </div>
+    </header>
+    <section class="modal-card-body">
+      <div *ngIf="showSpinner" class="app-loading">
+        &nbsp;
+        <svg class="spinner" viewBox="0 0 100 100">
+          <circle class="path" cx="50%" cy="50%" r="10" fill="none" stroke-width="2" stroke-miterlimit="10" />
+        </svg>
+      </div>
+      <div *ngIf='!showSpinner && showGraph' class="columns is-mobile">
```

Comment:
we should remove `showGraph` from here? 
As this is being set on `ReadingsValidator.validate(data); ` so if data for grpah is not numeric, allow user to select asset/sensor to fetch  series  for newly selected item